helm/kanister-operator/crds/repositoryserver.yaml

```diff
@@ -0,0 +1 @@
+../../../pkg/customresource/repositoryserver.yaml
```

Comment:
Let's not add any reposerver resources to helm chart as of now. Or have them behind a featureflag